### PR TITLE
replace z linux one pictos with z16

### PR DESCRIPTION
**Pull request by dudley-ibm**
**Removed**

- {{removed both `ibm--z-and-linuxone-multi-frame` and `ibm--z-and-linuxone-single-frame` pictograms from the Data Storage category since they are to be replaced by `ibm--z16--multi-frame` and `ibm--z16--single-frame` respectively, which were both added in PR #11932}}

#### Testing / Reviewing

{{ These removals should also be reflected on the IBM Design Language site }}

### CODE REVIEW

**Comment by tw15egan** [comment]
@tay1orjones can we remove these immediately, or do we need to deprecate them first?

**Comment by tay1orjones** [comment]
@tw15egan Yeah we can't delete these as they haven't been deprecated and removals would be a breaking change. This file needs updated instead:

https://github.com/carbon-design-system/carbon/blob/f26d4a0fd47ecf94d22e67c4cf14ad6cdefedb94/packages/icons/deprecated.yml#L1-L3

File: packages/elements/src/__tests__/__snapshots__/PublicAPI-test.js.snap [hunk 1]
```diff
@@ -305,7 +305,6 @@ Array [
   "textSecondary",
   "themes",
   "toggleOff",
-  "unstable_metadata",
```
Comment (tay1orjones) [inline packages/elements/src/__tests__/__snapshots__/PublicAPI-test.js.snap:308]: This doesn't look correct. I'd try updating snapshots again after pulling in latest from `main`

**Review by tay1orjones — CHANGES_REQUESTED** [review]
@tw15egan Yeah we can't delete these as they haven't been deprecated and removals would be a breaking change. This file needs updated instead:

https://github.com/carbon-design-system/carbon/blob/f26d4a0fd47ecf94d22e67c4cf14ad6cdefedb94/packages/icons/deprecated.yml#L1-L3

**Comment by tay1orjones** [comment]
Hey, I'm sorry - in trying to update this PR I really messed with the history and there's a bunch of uneccesary files. Going to close this one in favor of a new PR: https://github.com/carbon-design-system/carbon/pull/11973

**Pull request closed without merging**